packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -167,7 +173,8 @@ export const VBtn = defineComponent({
 
             link.navigate?.(e)
             group?.toggle()
-          } }
+          }}
```

Comment:
```suggestion
          } }
```

Reply:
is there a way for eslint (via prettier?) to automatically apply this formatting rule?

Reply:
I wish lol. For now it's just an internal team guideline.